app/controllers/solidus_paypal_commerce_platform/orders_controller.rb

```diff
@@ -1,14 +1,25 @@
 module SolidusPaypalCommercePlatform
   class OrdersController < ::Spree::Api::BaseController
     before_action :load_order
-    before_action :load_payment_method
+    before_action :load_payment_method, only: :create
     skip_before_action :authenticate_user
 
     def create
       authorize! :update, @order, order_token
       render json: @payment_method.gateway.create_order(@order, @payment_method.auto_capture), status: :ok
     end
 
+    def update_address
+      authorize! :update, @order, order_token
+      paypal_address = SolidusPaypalCommercePlatform::PaypalAddress.new(@order)
+
+      if paypal_address.update(params[:address])
```

Comment:
Wondering why this is not blocked by strong-parameters 🤔

Reply:
Good question. I think it's because it's not an AR method, it's a custom defined method on paypal_address. Nevertheless, I added paypal_order_params method that permits the relevant fields! Thanks! 🦾

Reply:
Right! I forgot how it's AR's responsibility to do the check 😅